label_studio/projects/models.py

```diff
@@ -781,8 +782,12 @@ def update_tasks_counters(self, queryset, from_scratch=True):
         total_annotations = Count("annotations", distinct=True, filter=Q(annotations__was_cancelled=False))
         cancelled_annotations = Count("annotations", distinct=True, filter=Q(annotations__was_cancelled=True))
         total_predictions = Count("predictions", distinct=True)
-        if isinstance(queryset, list):
+        # construct QuerySet in case of list of Tasks
+        if isinstance(queryset, list) and isinstance(queryset[0], Task):
```

Comment:
what if len(queryset)=0?

Reply:
It should, but added conditions to check length